doc/developers/contributing.rst

```diff
@@ -155,22 +155,12 @@ If any of the above seems like magic to you, then look up the `Git documentation
 <http://docs.scipy.org/doc/numpy/dev/gitwash/development_workflow.html>`_ on the
 web.
 
-If some conflicts arise between your branch and the ``master`` branch, you need
-to merge ``master``. The command will be::
-
-  $ git merge master
-
-with ``master`` being synchronized with the ``upstream``.
-
-Subsequently, you need to solve the conflicts. You can refer to the `Git
-documentation related to resolving merge conflict using the command line
-<https://help.github.com/articles/resolving-a-merge-conflict-using-the-command-line/>`_.
-
-.. note::
-
-   In the past, the policy to resolve conflicts was to rebase your branch on
-   ``master``. GitHub interface deals with merging ``master`` better than in
-   the past.
+In particular, if some conflicts arise between your branch and the master
+branch, you will need to `rebase your branch on master
+<http://docs.scipy.org/doc/numpy/dev/gitwash/development_workflow.html#rebasing-on-master>`_.
+Please avoid merging master branch into yours. If you did it anyway, you can fix
+it following `this example
```

Comment:
There something with your branch. You are not synchronized. I made changes in PR #8753 and you should not have the previous version here.